# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/PHttpChannel.ipdl
# Commit: f75cc7d5f40e
# Full Hash: f75cc7d5f40e5973936997f7a0f66707860c9be0
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-30 10:00:42
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P11 Move classification IPC methods back to PHttpChannel. r=mayhemer,necko-reviewers
#   
#   Bug 1015466 move them from PContent to Pbg since it believes those events are fine
#   to received before OnStopRequest. Bug 1588241 move it back to PContent since those
#   events are before OnStartRequest.
# ==============================================================================

diff -r 2289e685031e -r f75cc7d5f40e netwerk/protocol/http/PHttpChannel.ipdl
--- a/netwerk/protocol/http/PHttpChannel.ipdl	Tue Jun 30 00:34:06 2020 +0000
+++ b/netwerk/protocol/http/PHttpChannel.ipdl	Tue Jun 30 00:34:24 2020 +0000
@@ -14,12 +14,9 @@
 include IPCServiceWorkerDescriptor;
 include IPCStream;
 include HttpChannelParams;
-include PURLClassifierInfo;
 
 include "mozilla/net/NeckoMessageUtils.h";
 
-using nsIHttpChannel::FlashPluginState from "mozilla/net/NeckoMessageUtils.h";
-
 namespace mozilla {
 namespace net {
 
@@ -160,18 +157,6 @@
 
   async AltDataCacheInputStreamAvailable(IPCStream? stream);
 
-  // Tell the child that the resource being loaded has been classified.
-  async NotifyClassificationFlags(uint32_t aClassificationFlags, bool aIsThirdParty);
-
-  // Tell the child that the current channel's document is not allowed to load
-  // flash content.
-  async NotifyFlashPluginStateChanged(FlashPluginState aState);
-
-  // Tell the child information of matched URL againts SafeBrowsing list
-  async SetClassifierMatchedInfo(ClassifierInfo info);
-
-  // Tell the child information of matched URL againts SafeBrowsing tracking list
-  async SetClassifierMatchedTrackingInfo(ClassifierInfo info);
 
 both:
   // After receiving this message, the parent also calls
